brozzler/__init__.py

```diff
@@ -321,10 +321,9 @@ def _remove_query(url):
 # XXX chop off path after last slash??
 site_surt_canon = urlcanon.Canonicalizer(urlcanon.semantic.steps + [_remove_query])
 
-import doublethink
```

Comment:
We still end up transiently pulling in doublethink from importing `brozzler.worker`, `brozzler.model` and functions from `brozzler.cli`. We need to do something like #334 to gate off importing anything from those models if `import doublethink` is unavailable. eg, with this branch and doublethink not available:

```
>>> import brozzler
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/mistydemeo/git/brozzler/brozzler/__init__.py", line 331, in <module>
    from brozzler.worker import BrozzlerWorker
  File "/Users/mistydemeo/git/brozzler/brozzler/worker.py", line 36, in <module>
    import doublethink
ModuleNotFoundError: No module named 'doublethink'
```